pallets/sidechain/Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 name = "pallet-sidechain"
-description = "Pallet for integritee sidechains"
+description = "Pallet for litentry sidechains"
 version = "0.9.0"
-authors = ["Integritee AG <hello@integritee.network>"]
-homepage = "https://integritee.network/"
-repository = "https://github.com/integritee-network/pallets/"
+authors = ['Litentry Technologies GmbH <info@litentry.com> and Integritee AG <hello@integritee.network>']
```

Comment:
This is already an array so maybe `['Litentry Technologies GmbH <info@litentry.com>', 'Integritee AG <hello@integritee.network>']` ?

Reply:
Thanks, this is indeed cleaner